pheweb/utils.py

```diff
@@ -48,6 +48,19 @@ def get_phenolist():
         pheno['phenocode'] = urllib.parse.quote_plus(pheno['phenocode'])
     return phenolist
 
+def get_use_phenos():
+    from .file_utils import common_filepaths
+    filepath = common_filepaths['use_phenos']
+    try:
+        with open(os.path.join(filepath), encoding='utf-8') as f:
+            phenolist = [pheno.strip() for pheno in f.readlines() if pheno != '']
```

Comment:
make it dict for efficient lookup

Reply:
It becomes a dict but for consistency is a list here

Reply:
oh so it does good